lib/ig.js

```diff
@@ -824,10 +826,42 @@ ${match[1]}
         };
         // Add the resource to the XML
         pushXmlResource(`${example.resourceType}/${example.id}`, `Example: ${example.id}`, 'example');
+        examplesList.push(example);
       } catch (e) {
         console.warn('Invalid example.  Example must be valid JSON:', exFilePath, e);
       }
     }
+    if (examplesList.length > 0) {
+      // Add the example to the HTML
+      examplesList.sort((a, b) => a.resourceType.localeCompare(b.resourceType));
+      for (const example of examplesList) {
+        let trimmedProfiles = [];
+        if (example.meta != null) {
+          // Get profile names trimmed down to just namespace-name
+          trimmedProfiles = example.meta.profile.map(profile => profile.slice(profile.lastIndexOf('/') + 1));
+        }
+        // Build up a set of links to profiles
+        const profileStrings = trimmedProfiles.map(profile => {
+          // If profile is internal to the IG, link to it. Otherwise, do not give a link
+          const linkableProfiles = fhirResults.profiles.map(p => p.id);
+          if (linkableProfiles.indexOf(profile) > -1) {
```

Comment:
Comparing on the id isn't really sufficient, as profile IDs are not required to be unique across IGs.  It's not likely to have a clash in our case, since we encode the namespace into the ID -- but we've actually had some requests to *stop* doing that, so we shouldn't rely on it.

It would be better to use the URLs for the match (the URL in the meta.profile of the example, and the "url" property of the profile.

Reply:
Fixing this caused me to also refactor the code slightly, changes are here https://github.com/standardhealth/shr-fhir-export/pull/164/commits/503bb7550aeff0ae928de9f8fbcd4d444e4e09f5.